healthcareai/common/transformers.py

```diff
@@ -54,7 +56,17 @@ def transform(self, X, y=None):
         # Return if not imputing
         if self.impute is False:
             return X
-
+        
+        #Warn users if %nan is too high
```

Comment:
I'd be thrilled if this were factored out to a separate function with tests (and I'm happy to guide you through that if you'd like).